reference/configuration/framework.rst

```diff
@@ -1560,6 +1561,16 @@ paths
 This option allows to define an array of paths where the component will look
 for translation files.
 
+.. _reference-translator-default_path:
+
+default_path
+.....
+
+**type**: ``string`` **default**: ``%kernel.project_dir%/translations``
```

Comment:
When documenting a new feature, we put a warning about the Symfony version where it was introduced. Could you please put this before this line? Thanks!

```rst
default_path
............

.. versionadded:: 3.4
    The ``default_path`` option was introduced in Symfony 3.4.

**type**: ``string`` **default**: ...
```